docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -405,6 +405,13 @@ class Sidebar extends React.Component<any, any> {
         to: '/important-and-mention-messages',
         styles: menuItemStyles,
       },
+      {
+        key: 'message-reactions-with-popup',
```

Comment:
sidebar becomes pretty full of prototypes, can we try grouping chat related prototypes by context the same we are doing for:

- dropdowns: http://localhost:8080/prototype-dropdowns
- popups: http://localhost:8080/prototype-popups

Reply:
Will do, for design yep, we decided because of accessibility reasons to replace the List with Menu, so it is ugly at this moment. Will address it, thanks!